src/libraries/System.Net.Sockets/src/System/Net/Sockets/SafeSocketHandle.cs

```diff
@@ -50,6 +50,14 @@ public SafeSocketHandle(IntPtr preexistingHandle, bool ownsHandle)
 
         internal bool OwnsHandle { get; }
 
+        internal bool HasShutdownSend
```

Comment:
Nit, we prefer => (expression bodied member) for a one line getter like this.

Reply:
Alright, I'll keep it in my mind! Changing it.